plugin-feature-doc.md

```diff
@@ -0,0 +1,111 @@
+# Plugin API is simple and well-documented
+
+The current plugin API has an overly large surface area, poor documentation,
+and unclear names.  The objective is to improve this, focusing on the UX of
+primary use-case: creating new service authenticators.
+
+Aha Card: https://cyberark.aha.io/features/AAM-92
```

Comment:
It's not clear to me whether this is correct - I've asked PM for clarification, but I haven't heard back.

I think AAM-92 was revised to refer only to the configuration refactoring we completed in #707. I'm not sure there is an Aha card for this, since it's engineering driven work.